addonscript/exceptions.py

```diff
@@ -1,7 +1,15 @@
 """addonscript specific exceptions."""
 
+from scriptworker.exceptions import ScriptWorkerRetryException
 
-class SignatureError(Exception):
+
+class SignatureError(ScriptWorkerRetryException):
+    """Error when signed XPI is still missing or reported invalid by AMO."""
+
+    pass
+
+
+class FatalSignatureError(Exception):
```

Comment:
We may want to use ScriptWorkerTaskException or other scriptworker exception, which sets the exit code so we die with the appropriate taskcluster status (failure / which exception reason).

Reply:
I like it, but deferring. (default scriptworker exit code is used here anyway)